tests/test_json_formatter.py

```diff
@@ -99,6 +99,20 @@ def test_date_format(self):
             else:
                 self.assertRegexpMatches(content['asctime'], pattern)
 
+    def test_date_format_epoch(self):
+        formatter = JsonFormatter(datefmt='U')
+        log = self.get_logger(formatter)
+        log.info("test a date in Unix epoch format")
+        with open(self.filename) as f:
+            content = json.loads(f.readlines()[-1])
+            pattern = '^\d{10}$'
+            if sys.version_info >= (3, 1):  # Python version >= 3.1
+                print(type(content['asctime']))
+                self.assertIsInstance(content['asctime'], int)
+                self.assertRegex(str(content['asctime']), pattern)
```

Comment:
If you are checking that time is represented as an integer, so no need to use a regexp for further tests. Just check a length. It will work on any Python version.

Reply:
Done